src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -282,4 +282,29 @@ private function applyCurrentStyle(string $text, string $current, int $width, in
 
         return implode("\n", $lines);
     }
+
+    private function addLineBreaks(string $text, int $width): string
```

Comment:
just wondering: can't we use wordwrap method provided by the String component?

Reply:
Yes we could, bit this would change the general behavior of the output a bit. Wordwrap tries to not break words and makes the linebreak before the word. Only if the word is longer than the length of the line there is the option to break the word.

```
wordwrap('test me in new line',12,"\n"); // result: "test me in \nnew line"; in console solution: "test me in n\new line"
wordwrap('thisIsLong',5,"\n",true); // result: "thisI\nsLong"
```

Reply:
Would work for me.
We just need to detect the encoding with mb_detect_encoding and do 
`b($s)->toCodePointString($encoding)->wordwrap()->toByteString($encoding)`

Reply:
I'll try that and see where all the console tests will fail :)